pkgs/development/compilers/sagittarius-scheme/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv
+, fetchurl
+, cmake
+, pkgconfig
+, libffi
+, boehmgc
+, openssl
+, zlib
+, odbc ? true
+, libiodbc
+}:
+
+let version = "0.9.6";
+    odbcDeps = if odbc then [ libiodbc ] else [];
```

Comment:
This line is not required. See `buildInputs` comment below.

Reply:
Done.